.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: 💡 Feature Request
+description: Suggest an idea for EmbassyOS
+title: '[feat]: '
+labels: ["Bug", "Needs Triage"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Please confirm you have completed the following.
+      options:
+        - label: I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already report this problem, without success.
```

Comment:
This also may need rewording to reflect that they have searched for the feature request already.  There is also a section for this in our Discussions, I believe